intro-HPC/gent/macros.tex

```diff
@@ -3,20 +3,22 @@
 \macro{\university}{UGent}
 \macro{\association}{AUGent}
 % HPC definitions
-\macro{\hpc}{HPC-UGent}
+\macro{\hpc}{UGent-HPC}
```

Comment:
"the High Performance Computing" doesn't make sense at all, whether you use "UGent" before or after...

"the HPC-UGent infrastructure" does make sense

we've been using "HPC-UGent" quite consistently, so don't use "UGent-HPC"